tests/firedrake/conftest.py

```diff
@@ -10,6 +10,83 @@
 from firedrake.petsc import PETSc, get_external_packages
 
 
+def _skip_test_dependency(dependency):
```

Comment:
Might it make more sense to rename this "`dependency_is_present(dep)`" or similar?

Reply:
See above.